pinot-spi/src/main/java/org/apache/pinot/spi/config/table/FieldConfig.java

```diff
@@ -51,11 +52,13 @@
   public FieldConfig(@JsonProperty(value = "name", required = true) String name,
       @JsonProperty(value = "encodingType") @Nullable EncodingType encodingType,
       @JsonProperty(value = "indexType") @Nullable IndexType indexType,
-      @JsonProperty(value = "properties") @Nullable Map<String, String> properties) {
+      @JsonProperty(value = "properties") @Nullable Map<String, String> properties,
+      @JsonProperty(value = "noDictionaryColumnCompressionCodec") @Nullable NoDictionaryColumnCompressionCodec noDictionaryColumnCompressionCodec) {
```

Comment:
Shall we simplify the field name, e.g. `compressionCodec`? This long name is slightly hard to config, and we don't need to separate the codec of raw vs dictionary encoded

Reply:
`compressionCodec` works, will update!

Reply:
done!